build/build-treemap.js

```diff
@@ -15,8 +17,11 @@ const GhPagesApp = require('./gh-pages-app.js');
  */
 function buildStrings() {
   const locales = require('../lighthouse-core/lib/i18n/locales.js');
-  const UIStrings = require('../lighthouse-treemap/app/src/util.js').UIStrings;
-  const strings = /** @type {import('../lighthouse-treemap/types/treemap').Strings} */ ({});
+  const UIStrings = require(
+    // Prevent tsc from evaluating util.js
```

Comment:
logger issues were from the Strings import.


util.js import causes this: 

![image](https://user-images.githubusercontent.com/4071474/119014538-eecd0a00-b94c-11eb-9b25-199e633df44e.png)

util.js is too important to ts-nocheck

Reply:
Hm, then I don't understand the code comment 😅  `tsc` *is* evaluating `util.js` or it isn't?

Sorry I'm just not missing how it can simultaneously be too important to not type check **and** we're actively trying to prevent tsc from evaluating it.

Reply:
Prevent `tsc -p .` from evaluating it...still gets processed for `tsc -p lighthouse-treemap`.

I don't know what I'm doing here, this is a bandaid, if this matters then @brendankenny will come to the rescue :)

Reply:
Gotcha. Something like this then?
```suggestion
    // Prevent `tsc -p .` from evaluating util.js using core types, it is already typchecked by `tsc -p lighthouse-treemap`
```